### ISSUE TITLE
Downloads 16:9 thumbnails from YouTube Music and uses " - Topic" uploader name

### ISSUE THREAD

**Issue by SpectralKH** [rg3/youtube-dl#18511]
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.12.09**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [x] Bug report, at least kind of

---

Downloading from YouTube Music results in a 16:9 thumbnail and an uploader name ending with ` - Topic`.

Here's an example where I run the command `youtube-dl -x --audio-format mp3 -o '%(uploader)s.%(ext)s' --embed-thumbnail https://music.youtube.com/watch?v=uRGgWKEiOHg`.
The URL's page looks like this:
![image](https://user-images.githubusercontent.com/11315492/49955603-7eac3580-ff03-11e8-8ca8-e58fdaeb37eb.png)
This is the thumbnail that is added to the output file:
![flutter](https://user-images.githubusercontent.com/11315492/49955670-b4511e80-ff03-11e8-9e4a-2c8baa8ba8b7.jpg)
This could be fine for a YouTube URL, but in this case the URL's page does not contain this thumbnail anywhere.
Additionally, the output file ends up with the filename `Diamond Eyes - Topic.mp3`. Again, this might be ok for a regular YouTube URL, but in the URL there is no sign of " - Topic" being part of the uploader's name.

**Comment by ealgase** [comment]
music.youtube.com is just another client for YouTube (just like gaming.youtube.com).  If you check the video at https://www.youtube.com/watch?v=uRGgWKEiOHg, you'll see that both the thumbnail and uploader are correct.

**Comment by SpectralKH** [comment]
@ealgase That's why I'm saying it's fine for YouTube URLs but not YouTube Music URLs. Either way it's not good UX

**Comment by ealgase** [comment]
@SpectralKH true.  The way to fix this is to make a separate extractor for YouTube music, that gets the thumbnail and artist name, then sends gets the normal YouTube extractor to get the rest of the data.
